flowpipe/node.py

```diff
@@ -473,6 +508,11 @@ def _initialize(self, func, outputs, metadata):
             for output in outputs:
                 OutputPlug(output, self)
 
+    def to_pickle(self):  # pragma: no cover
+        """Pickle the node. -- DOES NOT WORK FOR FunctionNode!"""
+        raise NotImplementedError("Pickling is not implemented for "
```

Comment:
Maybe we should add something like: "If you want to use pickle with this node, consider converting your Node to a class subclassing directly from flowpipe.node.INode"